GVFS/FastFetch/NativeUnixMethods.cs

```diff
@@ -80,6 +94,9 @@ public static bool TryStatFileAndUpdateIndex(ITracer tracer, string path, Memory
         [DllImport("libc", EntryPoint = "write", SetLastError = true)]
         private static unsafe extern IntPtr Write(int fileDescriptor, void* buf, IntPtr count);
 
+        [DllImport("libc", EntryPoint = "symlink", SetLastError = true)]
+        private static extern int CreateSymLink(string sourcePath, string destinationPath);

```

Comment:
Naming suggestion: The meanings of `sourcePath` and `destinationPath` are a bit vague when it comes to symlinks.  Perhaps something like `linkTarget` and `newLinkPath` instead?